Tests/SunCalcTests.swift

```diff
@@ -39,15 +39,6 @@ class SunCalcTests: XCTestCase {
 
         let sunPhaseTimes = SunTime(date: date, coordinate: coordinate).sunTimes
 
-        // DAY 4
-        dateComponents.day = 4
-
-        dateComponents.hour = 22
-        dateComponents.minute = 10
-        dateComponents.second = 57
-        let expectedNadir = calendar.date(from: dateComponents)!
-        XCTAssertEqual(sunPhaseTimes.nadirTime.timeIntervalSince1970, expectedNadir.timeIntervalSince1970, accuracy: 0.99)
-
```

Comment:
why are these various cases being deleted?

Reply:
I deleted them in the previous refactor branch

Reply:
"them" meaning what?

Reply:
those various cases :) `nadirTime` and whatnot

Reply:
So is `nadirTime` gone from the app now? i'm poking at why these tests are no longer in here: Are they no longer relevant, or were they just not passing?

Reply:
They're no longer relevant, these times were part of the SunPhase enum, but were never used, so i removed them.

Reply:
Great, got it. Thank you

Reply:
Where it happend: https://github.com/bakkenbaeck/daylight-ios/pull/118/commits/4071e08a597e434879c56da9531ac18e6367d83d